cypress.json

```diff
@@ -5,6 +5,6 @@
   "video": true,
   "retries": 2,
   "chromeWebSecurity": false,
-  "viewportWidth": 1000,
-  "viewportHeight": 600
+  "viewportWidth": 360,
+  "viewportHeight": 640
```

Comment:
I liked this change at first (LH's Moto G4 dimensions), but we should remember that the integration tests from WebOps CI won't use these values and will keep using 1000 x 600 so there could be some minor results in local/CI runs. 

Ideally, there shouldn't be many differences, but apparently, there are some. I think the manual position adjustments we needed to make to the infinite scroll tests were considering the 1000 x 600 in mind.